presentation/src/main/java/com/wednesday/template/presentation/weather/start/StartScreenViewModel.kt

```diff
@@ -1,22 +1,35 @@
 package com.wednesday.template.presentation.weather.start
 
+import com.wednesday.template.interactor.weather.SearchCityInteractor
+import com.wednesday.template.navigation.search.SearchNavigator
+import com.wednesday.template.presentation.base.UIText
+import com.wednesday.template.presentation.base.UIToolbar
 import com.wednesday.template.presentation.base.intent.IntentHandler
 import com.wednesday.template.presentation.base.viewmodel.BaseViewModel
 
-class StartScreenViewModel :
-    BaseViewModel<
-        StartFragmentScreen, StartScreenState>(),
+class StartScreenViewModel(
+    private val searchCityInteractor: SearchCityInteractor,
+    private val navigator: SearchNavigator
+) : BaseViewModel<StartFragmentScreen, StartScreenState>(),
     IntentHandler<StartScreenIntent> {
-
+    
     override fun getDefaultScreenState(): StartScreenState {
-        TODO()
+        return StartScreenState(
+            toolbar = UIToolbar(
+                title = UIText { block("Search") },
+                hasBackButton = true,
```

Comment:
This is the start page. This should not have a back button.

Reply:
Done